--- Uranium-2.5.0/CMakeLists.txt~	2017-05-13 23:51:30.000000000 +0200
+++ Uranium-2.5.0/CMakeLists.txt	2017-05-13 23:56:38.423247507 +0200
@@ -64,6 +64,6 @@
 install(FILES ${CMAKE_SOURCE_DIR}/cmake/UraniumTranslationTools.cmake
         DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/Modules/ )
 install(DIRECTORY resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
-install(DIRECTORY plugins DESTINATION lib/uranium)
+install(DIRECTORY plugins DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
 
 include(CPackConfig.cmake)
--- Uranium-2.5.0/UM/Application.py~	2017-04-13 15:03:28.000000000 +0200
+++ Uranium-2.5.0/UM/Application.py	2017-05-14 11:25:56.345442852 +0200
@@ -95,7 +95,7 @@
 
         self._plugin_registry = PluginRegistry.getInstance()
 
-        self._plugin_registry.addPluginLocation(os.path.join(Application.getInstallPrefix(), "lib", "uranium"))
+        self._plugin_registry.addPluginLocation(os.path.join(Application.getInstallPrefix(), "share", "uranium"))
         self._plugin_registry.addPluginLocation(os.path.join(os.path.dirname(sys.executable), "plugins"))
         self._plugin_registry.addPluginLocation(os.path.join(Application.getInstallPrefix(), "Resources", "uranium", "plugins"))
         self._plugin_registry.addPluginLocation(os.path.join(Application.getInstallPrefix(), "Resources", self.getApplicationName(), "plugins"))
