--- Uranium-2.5.0/CMakeLists.txt~	2017-05-13 23:51:30.000000000 +0200
+++ Uranium-2.5.0/CMakeLists.txt	2017-05-13 23:56:38.423247507 +0200
@@ -64,6 +64,6 @@
 install(FILES ${CMAKE_SOURCE_DIR}/cmake/UraniumTranslationTools.cmake
         DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/Modules/ )
 install(DIRECTORY resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
-install(DIRECTORY plugins DESTINATION lib/uranium)
+install(DIRECTORY plugins DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
 
 include(CPackConfig.cmake)
