diff -urNp -x '*.orig' Uranium-4.5.0.org/cmake/UraniumTests.cmake Uranium-4.5.0/cmake/UraniumTests.cmake
--- Uranium-4.5.0.org/cmake/UraniumTests.cmake	2020-02-24 16:02:05.000000000 +0100
+++ Uranium-4.5.0/cmake/UraniumTests.cmake	2021-03-03 12:41:43.155972843 +0100
@@ -48,9 +48,3 @@ foreach(_plugin ${_plugins})
         uranium_add_test(NAME pytest-${_plugin_name} DIRECTORY ${_plugin_directory} PYTHONPATH "${CMAKE_SOURCE_DIR}|${_plugin_directory}")
     endif()
 endforeach()
-
-#Add code style test.
-add_test(
-    NAME "code-style"
-    COMMAND ${PYTHON_EXECUTABLE} run_mypy.py WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-)
\ No newline at end of file
